curriculum/challenges/english/01-responsive-web-design/css-picasso-painting/part-060.md

```diff
@@ -0,0 +1,291 @@
+---
+id: 60b69a66b6ddb80858c515b1
+title: Part 60
+challengeType: 0
+dashedName: part-60
+---
+
+# --description--
+
+Within the `guitar` element, create three `div` elements. Give the first two a `class` value of `guitar`. Then give the first an `id` of `guitar-left`, and the second an `id` of `guitar-right`. Add an `id` to the third `div` with the value `guitar-neck`.
+
+The third `div` should not have the `guitar` class.
+
+# --hints--
+
+You should have three new `div` elements within your `guitar` element.
+
+```js
+assert(document.querySelectorAll('#guitar > div')?.length === 3);
+```
+
+Your first new `div` should have a `class` set to `guitar`.
+
+```js
+assert(document.querySelectorAll('#guitar > div')?.[0]?.classList?.contains('guitar'));
+```
+
+Your first new `div` should have an `id` set to `guitar-left`.
+
+```js
+assert(document.querySelectorAll('#guitar > div')?.[0]?.getAttribute('id') === 'guitar-left');
+```
+
+Your second new `div` should have a `class` set to `guitar`.
+
+```js
+assert(document.querySelectorAll('#guitar > div')?.[1]?.classList?.contains('guitar'));
+```
+
+Your second new `div` should have an `id` set to `guitar-right`.
+
+```js
+assert(document.querySelectorAll('#guitar > div')?.[1]?.getAttribute('id') === 'guitar-right');
+```
+
+Your third new `div` should have an `id` set to `guitar-neck`.
+
+```js
+assert(document.querySelectorAll('#guitar > div')?.[2]?.getAttribute('id') === 'guitar-neck');
+```
+
+# --seed--
```

Comment:
````suggestion
```

You should not give the third new `div` a `class` of `guitar`.

```js
assert.notExists(document.querySelector('#guitar > #guitar-neck.guitar'));
```

# --seed--
````